src/app/coursegrab/controllers/untrack_course_controller.py

```diff
@@ -0,0 +1,20 @@
+from . import *
+
+
+class UntrackCourseController(AppDevController):
+    def get_path(self):
+        return "/users/untrack/"
+
+    def get_methods(self):
+        return ["DELETE"]
```

Comment:
prefer post here tbh, if it were a delete it would be on /subscriptions/ or something but that's too clunky